packages/grafana-ui/.storybook/preview.ts

```diff
@@ -38,6 +38,7 @@ addParameters({
     light: GrafanaLight,
   },
   options: {
+    theme: GrafanaLight,
```

Comment:
Oops, this one slipped by. It's the same as https://github.com/grafana/grafana/pull/23383